README.md

```diff
@@ -175,6 +175,22 @@ export class AddSubscriptionController {
   ) {}
 ```
 
+## Use injectableSingleton decorator
+
+> If you want to inject the class using Singleton, this is a very simple approach:
+
+```typescript
+import { injectable } from "inversify";
+import { injectMapper, injectableSingleton } from "inversify-mapper";
```

Comment:
Why is it still necessary to import injectable from the "inversify" path?

Reply:
Its not necessary. Fixed readme.